src/components/organisms/SectionContents/individual.js

```diff
@@ -6,20 +6,26 @@ import * as React from 'react'
 import ModalBtn from "../../atoms/ModalButton";
 import Alert from "../../atoms/Alert";
 
+//** Helpers */
+//** Personalization */
+import Text from "../../helper/Text";
+import { renderToString } from 'react-dom/server';
+import ReactHtmlParser from 'react-html-parser'; 
+
 /**
  * Section: Individual
  */
```

Comment:
This is disgusting drug-induced crap, and should die
To put it with Linus words.